test/functional/services/dashboard/visualizations.js

```diff
@@ -56,8 +55,6 @@ export function DashboardVisualizationProvider({ getService, getPageObjects }) {
       }
 
       await PageObjects.discover.saveSearch(name);
-      await PageObjects.header.waitUntilLoadingHasFinished();
-      await testSubjects.exists('saveSearchSuccess');
```

Comment:
how come no replacement with the dismissing of toast here?

Reply:
oh nm you probably moved it into `saveSearch`, haven't gotten there yet.

Reply:
Right, `PageObjects.discover.saveSearch()` now checks for and closes the toast.